### PR TITLE
Meilisearch Pagination, API-Key and Searchindex

### PR DESCRIPTION
**Pull request by chripoh**
- add named constant ROTARACT_MEILISEARCH_SEARCHINDEX to switch Meilisearch index
- add pagination parameter 'limit' to Meiliseach Filter, to tackle Ticket#3514348 (set limit: 25)
- changed named constant ROTARACT_MEILISEARCH_API_KEY to ROTARACT_MEILISEARCH_CLUBFINDER_API_KEY

### CODE REVIEW

**Comment by bbenno** [comment]
LGTM, can you add some change notes to the CHANGELOG?

**Comment by chripoh** [comment]
> LGTM, can you add some change notes to the CHANGELOG and update the README as well?

Sure, I can do that, but I still have a question:
Since I have renamed Named Constant, it is no longer backwards compatible, so after Semantic Versioning 2.0.0 it would be a new MAJOR release, in this case 5.0.0. Right?

**Comment by bbenno** [comment]
> in this case 5.0.0. Right?

correct :)

**Comment by chripoh** [comment]
The Changelog points to the Rotaract Appointments Plugin. 
Is this intended?

```
= Changelog

:repo: https://github.com/rotaract/rotaract-appointments
:compare: {repo}/compare
:github-pr: {repo}/pull
[...]
```

**Comment by bbenno** [comment]
Whoops, no it is not. Would appreciate if you fix it :)

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 
 The format is based on https://keepachangelog.com/en/1.0.0/[Keep a Changelog], and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Versioning].
 
+== {compare}/Unreleased\...main[5.0.0]
+
+=== Changed
+
+* add new named constant ROTARACT_MEILISEARCH_SEARCHINDEX
+* changed named constant ROTARACT_MEILISEARCH_API_KEY to ROTARACT_MEILISEARCH_CLUBFINDER_API_KEY
+* updated Meilisearch Filter
```
Comment (bbenno) [inline CHANGELOG.adoc:17]: better be more precise ;-)

**Review by bbenno — CHANGES_REQUESTED** [review]

**Review by bbenno — APPROVED** [review]
many thanks!

**Pull request merged**